pkgs/development/tools/refurb/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "refurb";
+  version = "1.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "dosisod";
+    repo = "refurb";
+    rev = "v${version}";
+    hash = "sha256-bWhIt91wGCOlknSVd12OgCKsPUw8iS3OSiKnsspxzPI";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    mypy
+    mypy-extensions
+    tomli
+    typing-extensions
+  ];
+
+  checkInputs = with python3Packages; [
+    pytest
+    attrs
+    black
```

```suggestion
    attrs
```
the build should fail with two pytests.